releases/release-1.13/release-notes-draft.md

```diff
@@ -422,4 +422,43 @@ SIG Windows focused on improving reliability for Windows and Kubernetes support
 - Handle Windows named pipes in host mounts. ([#69484](https://github.com/kubernetes/kubernetes/pull/69484), [@ddebroy](https://github.com/ddebroy))
 - Fixed inconsistency in windows kernel proxy when updating HNS policy. ([#68923](https://github.com/kubernetes/kubernetes/pull/68923), [@delulu](https://github.com/delulu))
 
-
+##  External Dependencies
+
+- Default etcd server was updated to v3.2.24. ([#68318](https://github.com/kubernetes/kubernetes/pull/68318))
+- Rescheduler is unchanged from v1.11: v0.4.0. ([#65454](https://github.com/kubernetes/kubernetes/pull/65454))
+- The list of validated docker versions was updated to 1.11.1, 1.12.1, 1.13.1, 17.03, 17.06, 17.09, 18.06. ([#68495](https://github.com/kubernetes/kubernetes/pull/68495))
+- The default Go version was updated to 1.10.4. ([68802](https://github.com/kubernetes/kubernetes/pull/68802))
+- The minimum supported Go version was updated to 1.10.2 ([#63412](https://github.com/kubernetes/kubernetes/pull/63412))
+- CNI is unchanged from v1.10: v0.6.0 ([#51250](https://github.com/kubernetes/kubernetes/pull/51250))
+- CSI is unchanged from v1.11:  0.3.0 ([#64719](https://github.com/kubernetes/kubernetes/pull/64719))
+- The dashboard add-on unchanged from v1.10: v1.8.3. ([#57326](https://github.com/kubernetes/kubernetes/pull/57326))
+- Bump Heapster to v1.6.0-beta as compared to v1.5.2 in v1.11  ([#67074](https://github.com/kubernetes/kubernetes/pull/67074))
+- Cluster Autoscaler has been upgraded to v1.12.0 ([#s8739](https://github.com/kubernetes/kubernetes/pull/68739))
+- kube-dns was updated to v1.14.13. ([#68900](https://github.com/kubernetes/kubernetes/pull/68900))
+- Influxdb is unchanged from v1.10: v1.3.3 ([#53319](https://github.com/kubernetes/kubernetes/pull/53319))
+- Grafana is unchanged from v1.10: v4.4.3 ([#53319](https://github.com/kubernetes/kubernetes/pull/53319))
+- Kibana is at v6.3.2.  ([#67582](https://github.com/kubernetes/kubernetes/pull/67582))
+- CAdvisor is unchanged from v1.11:  v0.30.1 ([#64987](https://github.com/kubernetes/kubernetes/pull/64987))
+- fluentd-gcp-scaler has been updated to v0.4.0, up from 0.3.0 in v1.11. ([#67691](https://github.com/kubernetes/kubernetes/pull/67691))
+- fluentd in fluentd-es-image is unchanged from 1.10: v1.1.0 ([#58525](https://github.com/kubernetes/kubernetes/pull/58525))
```

Comment:
this entry should be removed IMO.
the one bellow is the actual update value - 1.2.4.